### ISSUE TITLE
Reindexing of atoms causes problem in mapping

### ISSUE THREAD

**Issue by ppxasjsm** [CCPBioSim/fesetup#8]
I am not sure if this is intended, but in order to overcome poor mapping one way of doing so is giving an explicit mapping. if I use atom indexes from the input pdb file the explicit mapping will still fail, because FESetup changed the atom indexing in the parametrisation. So only once I have generated a set of vacuum.rst7 and vacuum.parm7 files is it possible to do explicit mapping with the atom indexes in these files. 

Input file before parametrisation with atom indexes:
![input_cats_29](https://user-images.githubusercontent.com/3668316/47649578-a2930200-db75-11e8-8d37-c6f925994211.png)

Atom indexes after parametrisation:
![parametrised_cats_29](https://user-images.githubusercontent.com/3668316/47649602-b0e11e00-db75-11e8-8442-22807a5f43ea.png)

**Comment by skfegan** [comment]
I don't think this is explicitly an FESetup problem. The reindexing is as far as I know happening in one of the dependencies, probably tleap or in a format conversion (from pdb to mol2 or the like).
